src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -41,10 +44,57 @@ public void Initialize(ExtensionConfigContext context)
             {
                 throw new ArgumentNullException("context");
             }
+
+            INameResolver nameResolver = context.Config.NameResolver;
+            IExtensionRegistry extensions = context.Config.GetService<IExtensionRegistry>();
             var converterManager = context.Config.GetService<IConverterManager>();
             converterManager = converterManager.AddNotificationHubConverters();
```

Comment:
The extension method to add converters? @pragnagopa can probably discuss that :-)

---
In reply to: [63599917](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63599917) [](ancestors = 63599917)